gslib/tests/test_naming.py

```diff
@@ -979,19 +979,56 @@ def testRecursiveCopyObjsAndFilesToExistingBucketSubDir(self):
   def testRecursiveCopyObjsAndFilesToNonExistentBucketSubDir(self):
     """Tests recursive copy of objs + files to non-existent bucket subdir."""
     src_bucket_uri = self.CreateBucket(test_objects=['f0', 'nested/f1'])
-    src_dir = self.CreateTempDir(test_files=['f2', ('nested', 'f3')])
+    src_dir = self.CreateTempDir(test_files=[('parent',
+                                              'f2'), ('parent', 'nested',
+                                                      'f3')])
```

Comment:
If you change this to `'f3'),])` (adding a comma after the tuple), I think yapf will format this in a slightly better way.